### DISCUSSION TITLE
How to create tiles animation in blender ?

### DISCUSSION THREAD

**Discussion by BuzzLEquerre** [flareteam/flare-game#977] [General]
Hello, i download "flare bridge" file in OpenGameArt, and I install Blender 2.58, and the problem is, when I want to create a Animation render in the FLARE-BridgeTall file I have a blackscreen. Idk why the another blender file works but if anyone has an idea to help me I would like this help :)

https://opengameart.org/content/isometric-bridge
![Uploading Capture d'écran 2024-11-10 145724.png…]()

**Comment by dorkster** [comment]
Do you mean that the background of the tiles is black? Make sure you have RGBA selected in the output settings:

![Screenshot_20241110_172601](https://github.com/user-attachments/assets/f6236b8b-0446-4bad-8fca-52503bb830e0)

**Reply by BuzzLEquerre** [reply]
ok it's fine I download your Blender version, V2.79 and I have the textures even if I need to replace the grassland texture
![Uploading Capture d'écran 2024-11-11 151713.png…]()
